apps/emqx/test/emqx_channel_SUITE.erl

```diff
@@ -1126,33 +1167,10 @@ mock_cm_open_session() ->
         end
     ).
 
-%% conn: 5/s; overall: 10/s
 quota() ->
-    emqx_limiter_container:get_limiter_by_types(?MODULE, [message_routing], limiter_cfg()).
-
-limiter_cfg() ->
-    Client = #{
-        rate => 5,
-        initial => 0,
-        burst => 0,
-        low_watermark => 1,
-        divisible => false,
-        max_retry_time => timer:seconds(5),
-        failure_strategy => force
-    },
-    #{
-        message_routing => bucket_cfg(),
-        client => #{message_routing => Client}
-    }.
-
-bucket_cfg() ->
-    #{rate => 10, initial => 0, burst => 0}.
-
-add_bucket() ->
-    emqx_limiter_server:add_bucket(?MODULE, message_routing, bucket_cfg()).
-
-del_bucket() ->
-    emqx_limiter_server:del_bucket(?MODULE, message_routing).
+    emqx_limiter_container:create_by_names(
+        [bytes, messages], #{}, emqx_limiter:internal_allocator()
```

Comment:
Maybe this function is not used?  There doesn't seem to be a `emqx_limiter:internal_allocator/0` function.  :thinking: